README.md

```diff
@@ -1,140 +1,111 @@
-  * [CommandKit::Commands::AutoRequire](https://rubydoc.info/gems/command_kit/CommandKit/Commands/AutoRequire)
-* [CommandKit::Description](https://rubydoc.info/gems/command_kit/CommandKit/Description)
-* [CommandKit::Env](https://rubydoc.info/gems/command_kit/CommandKit/Env)
-  * [CommandKit::Env::Home](https://rubydoc.info/gems/command_kit/CommandKit/Env/Home)
-  * [CommandKit::Env::Path](https://rubydoc.info/gems/command_kit/CommandKit/Env/Path)
-* [CommandKit::Examples](https://rubydoc.info/gems/command_kit/CommandKit/Examples)
-* [CommandKit::ExceptionHandler](https://rubydoc.info/gems/command_kit/CommandKit/ExceptionHandler)
-* [CommandKit::Help](https://rubydoc.info/gems/command_kit/CommandKit/Help)
-  * [CommandKit::Help::Man](https://rubydoc.info/gems/command_kit/CommandKit/Help/Man)
-* [CommandKit::Interactive](https://rubydoc.info/gems/command_kit/CommandKit/Interactive)
-* [CommandKit::Main](https://rubydoc.info/gems/command_kit/CommandKit/Main)
-* [CommandKit::OpenApp](https://rubydoc.info/gems/command_kit/CommandKit/OpenApp)
-* [CommandKit::Options](https://rubydoc.info/gems/command_kit/CommandKit/Options)
-  * [CommandKit::Options::Quiet](https://rubydoc.info/gems/command_kit/CommandKit/Options/Quiet)
-  * [CommandKit::Options::Verbose](https://rubydoc.info/gems/command_kit/CommandKit/Options/Verbose)
-* [CommandKit::OS](https://rubydoc.info/gems/command_kit/CommandKit/OS)
-  * [CommandKit::OS::Linux](https://rubydoc.info/gems/command_kit/CommandKit/OS/Linux)
-* [CommandKit::Pager](https://rubydoc.info/gems/command_kit/CommandKit/Pager)
-* [CommandKit::Printing](https://rubydoc.info/gems/command_kit/CommandKit/Printing)
-  * [CommandKit::Printing::Indent](https://rubydoc.info/gems/command_kit/CommandKit/Printing/Indent)
-* [CommandKit::ProgramName](https://rubydoc.info/gems/command_kit/CommandKit/ProgramName)
-* [CommandKit::Stdio](https://rubydoc.info/gems/command_kit/CommandKit/Stdio)
-* [CommandKit::Terminal](https://rubydoc.info/gems/command_kit/CommandKit/Terminal)
-* [CommandKit::Usage](https://rubydoc.info/gems/command_kit/CommandKit/Usage)
-* [CommandKit::XDG](https://rubydoc.info/gems/command_kit/CommandKit/XDG)
+* **Simplicity** - Commands are plain-old ruby classes, with options and arguments as attributes. Extend commands via inheritance. Uses [OptionParser][optparse] for POSIX option parsing.
+* **Subcommands** - Supports subcommands (explicit or lazy-loaded) and command aliases.
+* **Signals and errors** - Handles Ctrl^C and SIGINT interrupts (aka exit 130). Handles broken pipes (aka `mycmd | head`).
+* **Redirects** - Correctly handles when stdout or stdin is redirected to a file.
+* **Man** - Display a man-page instead of `--help` (see {CommandKit::Help::Man}).
+* **UX** - Optional ANSI coloring. Interactive prompt for user input. Supports easily detecting the terminal size, and paging output with `less` or `more`.
```

Comment:
Although it seems a little odd how we have concepts like Simplicity or UX along side things like Man and XDG. Maybe we should have a sub-list called "Standards" or "Integration" and list ANSI, XDG, etc.